lib/compiler.js

```diff
@@ -149,8 +149,8 @@ function findScripts(dir) {
 
   var results = [];
   files.forEach(function(filename) {
-    // ignore index.js and files prefixed with underscore
-    if ((filename === 'index.js') || (filename[0] === '_')) {
+    // ignore files prefixed with underscore
```

Comment:
I guess it's ok to include `custom-boot/script/index.js`. However, we should keep ignoring first-level index.js files like `custom-boot/index.js`, `boot/index.js`, etc. in order to preserve backward compatibility.